perllib/FixMyStreet/Cobrand/FixaMinGata.pm

```diff
@@ -87,43 +90,54 @@ sub geocoder_munge_results {
 
 sub area_types {
     my $self = shift;
+
     return $self->next::method() if FixMyStreet->staging_flag('skip_checks');
+
     [ 'KOM' ];
 }
 
 sub geocode_postcode {
     my ( $self, $s ) = @_;
+
     # Most people write Swedish postcodes like this:
     # XXX XX, so let's remove the space
     $s =~ s/\ //g;
+
     if ($s =~ /^\d{5}$/) {
         my $location = mySociety::MaPit::call('postcode', $s);
+
         if ($location->{error}) {
             return {
                 error => $location->{code} =~ /^4/
                     ? _('That postcode was not recognised, sorry.')
                     : $location->{error}
             };
         }
+
         return {
             latitude  => $location->{wgs84_lat},
             longitude => $location->{wgs84_lon},
         };
     }
+
     return {};
 }
 
 # Vad gör den här funktionen? Är "Sverige" rätt här?
 sub geocoded_string_check {
     my ( $self, $s ) = @_;
+
     return 1 if $s =~ /, Sverige/;
+
     return 0;
 }
 
 sub find_closest {
     my ( $self, $problem ) = @_;
+
     $problem = $problem->{problem} if ref $problem eq 'HASH';
-    return FixMyStreet::Geocode::OSM->closest_road_text( $self, $problem->latitude, $problem->longitude );
+
+    return FixMyStreet::Geocode::OSM::closest_road_text( $self, $problem->latitude, $problem->longitude );
```

Comment:
I don't understand the change here, this makes the tests fail and I think breaks, because it means the OSM class is no longer being passed as the first argument which was a change we made in 6c585965d612fa1ac89a2deeaf80ce3c2d8c8a2d. I've rebased and pushed a fixup so you can check this before merging.

Reply:
Oh, I see! Thanks for noticing this, and the fixup! This can be merged!